basic/src/App.css

```
@@ -0,0 +1,9 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+@layer components {
+  .card {
+    @apply rounded-xl border border-neutral-200 bg-white shadow;
```

```suggestion
    @apply rounded-xl border border-gray-200 bg-white p-6 shadow-sm transition-all duration-200;
```

This style of card will work with the more muted `bg-neutral-50` background color.

---

I just didn't add padding, since it can vary per component. Rest, yes sir!